client/verta/docs/tutorials/deployment/endpoint_config.rst

```diff
@@ -0,0 +1,89 @@
+Endpoint configuration
+======================
+
+Through an endpoint update, you can configure properties of the deployment including its
+autoscaling behavior and compute resources.
+
+Using the client
+----------------
+
+:meth:`Endpoint.update() <verta._deployment.endpoint.Endpoint.update>` provides parameters for
+configuring the resources and behavior of a deployed model. These settings can be used alongside
+any update strategy.
+
+.. code-block:: python
+
+    from verta.deployment.update import DirectUpdateStrategy
+
+    endpoint.update(
+        model_version, DirectUpdateStrategy(),
+        autoscaling=autoscaling,
+        resources=resources,
+        env_vars=env_vars,
+    )
+
+Autoscaling
+^^^^^^^^^^^
+
+``autoscaling`` takes an :class:`~verta.deployment.autoscaling._autoscaling.Autoscaling` object,
+which itself is used to establish upper and lower bounds for the number of replicas running the
+model. Autoscaling must also have at least one metric associated with it, which sets a threshold
+for triggering a scale-up.
+
+.. code-block:: python
+
+    from verta.deployment.autoscaling import Autoscaling
+    from verta.deployment.autoscaling.metrics import CpuUtilizationTarget
+
+    autoscaling = Autoscaling(max_replicas=4, min_scale=0.5)
+    autoscaling.add_metric(CpuUtilizationTarget(0.75))
+
+Here, CPU utilization exceeding 75% will lead to more replicas being created. For the full list of
+available metrics, see the :ref:`autoscaling` API documentation.
+
+Resources
+^^^^^^^^^
+
+``resources`` specifies the computational resources that will be available to the model when it is
+deployed.
+
+.. code-block:: python
+
+    from verta.deployment.resources import (CpuMillis, Memory)
+
+    resources = [
+        CpuMillis(250),
+        Memory("512Mi"),
+    ]
```

Comment:
This is confusing, right? What happens if I pass CPU twice?

Also, we need to document what `Mi` is, since that is _very_ implementation specific.

Reply:
>This is confusing, right? What happens if I pass CPU twice?
That's a great question! Would you suggest changing the Client API for resources? I think there's still time to do that. I'm imagining something like
```python
Resources(cpu_millis=250, memory="512Mi")
```

>Also, we need to document what Mi is, since that is very implementation specific.
ack; I had asked Evgenii to accept units based on https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-memory. I can copy the explanation from there.

Reply:
Something like that would work fine. I don't imagine we'll have too many resource options and we can split up if we do. But right now I don't know what happens if I violate what I imagine are constraints in the input.

Works for me. Probably we should have documented in the API itself.